docs/source/tutorials/image.ipynb

```diff
@@ -22,6 +22,56 @@
     "- Compute a list of potential label errors with cleanlab's `find_label_issues` method.\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "Quickstart\n",
+    "<br/>\n",
+    "    \n",
+    "Already have an sklearn compatible `model`, image `data` and true `labels`? Run the code below to train your `model` and get label issues.\n",
```

Comment:
"and true" -> "with given"

Reply:
We should make this ^ change for all notebooks, sorry I didn't catch it before!  Re terminology, we should be extremely wary about ever calling anything the "true label"